src/main/java/group1/mips_simulator/components/Register.java

```diff
@@ -5,6 +5,13 @@
  * the CPU to read/ write quickly.
  */
 public class Register {
+    regType type;
+    public Value value;
+
+    private enum regType{
+        GP, //General Purpose Registers
+        IX; //IndeX Registers
+    }
 
     public Value read() {
         // TODO
```

Comment:
What's the plan with these TODO statements? Are you planning on addressing these or is that a different task? It should be pretty easy right, just `return this.value;`? Or is there more complexity

Reply:
Same question wit the `set(...)` function

Reply:
yea, ill address those, just been putting together the skeleton, havent gotten down to implementing just yet.
was still processing the layout and project requirements, but ill be implementing these today